index.html

```diff
@@ -83,173 +83,213 @@ <h2 class="headerTitle greenText">I’m a software developer</h2>
-                <li class="tagText aboutTags">Capybara</li>
-                <li class="tagText aboutTags">Selenium</li>
+              <ul class="card-tags">
+                <li class="tag-text about-tags">Bootstrap</li>
+                <li class="tag-text about-tags">Ruby on rails</li>
+                <li class="tag-text about-tags">RSpec</li>
+                <li class="tag-text about-tags">Capybara</li>
+                <li class="tag-text about-tags">Selenium</li>
               </ul>
             </div>
           </li>
-          <li class="sectionCard aboutCard">
-            <div class="iconHolder">
+          <li class="section-card about-card">
+            <div class="icon-holder">
               <img src="asset/image//icons/langauge.png" alt="langauge icons" />
             </div>
-            <h3 class="sectionCardTitle">Skills</h3>
+            <h3 class="section-card-title">Skills</h3>
             <div>
-              <ul class="cardTags">
-                <li class="tagText aboutTags">Codekit</li>
-                <li class="tagText aboutTags">GitHub</li>
-                <li class="tagText aboutTags">Codepen</li>
-                <li class="tagText aboutTags">Gitlab</li>
-                <li class="tagText aboutTags">Terminal</li>
+              <ul class="card-tags">
+                <li class="tag-text about-tags">Codekit</li>
+                <li class="tag-text about-tags">GitHub</li>
+                <li class="tag-text about-tags">Codepen</li>
+                <li class="tag-text about-tags">Gitlab</li>
+                <li class="tag-text about-tags">Terminal</li>
               </ul>
             </div>
           </li>
         </ul>
       </section>
+      <section class="inner-section" id="contactSection">
+        <div class="form-container">
+          <div class="contact-intro">
+            <p class="section-title">
+              I'm always interested in hearing about new projects, so if you'd
+              like to chat please get in touch.
+            </p>
+          </div>
+          <div class="contact-form">
+            <form
+              action="https://formspree.io/f/xjvlpydr"
+              class="form"
+              method="post"
+            >
+              <input
+                type="text"
+                name="user_name"
+                id="user_name"
+                placeholder="Full name"
+              />
+              <input
+                type="email"
+                name="user_email"
+                id="user_email"
+                placeholder="Email address"
+              />
+              <textarea
+                name="message"
+                id="message"
+                cols="30"
+                rows="10"
+                placeholder="Enter Text Here"
+              ></textarea>
+              <button class="section-btn" type="submit" id="submit">
+                Get in Touch
+              </button>
+            </form>
```

Comment:
GG with your form.
-  [ ] As part of the requirements and to ensure a random user does not submit an empty form, I recommend adding `required` to each of your input field